crates/oxc_aria_query/src/lib.rs

```diff
@@ -0,0 +1,771 @@
+/// Auto generated by `tasks/a11y_data/src/lib.rs`.
+use phf::phf_set;
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
+pub struct Attr {
+    pub name: &'static str,
+    pub value: Option<&'static str>,
+}
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
+pub struct ElementSchema {
+    pub name: &'static str,
+    pub attributes: &'static [Attr],
+}
+pub static ABSTRACT_ROLES: phf::Set<&'static str> = phf_set! {
+    "command", "composite", "input", "landmark", "range", "roletype", "section",
+    "sectionhead", "select", "structure", "widget", "window",
+};
+
+pub static INTERACTIVE_ROLES: phf::Set<&'static str> = phf_set! {
+    "button", "checkbox", "columnheader", "combobox", "doc-backlink", "doc-biblioref",
+    "doc-glossref", "doc-noteref", "grid", "gridcell", "link", "listbox", "menu",
+    "menubar", "menuitem", "menuitemcheckbox", "menuitemradio", "option", "radio",
+    "radiogroup", "row", "rowheader", "scrollbar", "searchbox", "slider", "spinbutton",
+    "switch", "tab", "tablist", "textbox", "tree", "treegrid", "treeitem", "toolbar",
+};
+
+pub static NONINTERACTIVE_ROLES: phf::Set<&'static str> = phf_set! {
+    "alert", "alertdialog", "application", "article", "banner", "blockquote", "caption",
+    "cell", "code", "complementary", "contentinfo", "definition", "deletion", "dialog",
+    "directory", "doc-abstract", "doc-acknowledgments", "doc-afterword", "doc-appendix",
+    "doc-biblioentry", "doc-bibliography", "doc-chapter", "doc-colophon",
+    "doc-conclusion", "doc-cover", "doc-credit", "doc-credits", "doc-dedication",
+    "doc-endnote", "doc-endnotes", "doc-epigraph", "doc-epilogue", "doc-errata",
+    "doc-example", "doc-footnote", "doc-foreword", "doc-glossary", "doc-index",
+    "doc-introduction", "doc-notice", "doc-pagebreak", "doc-pagefooter",
+    "doc-pageheader", "doc-pagelist", "doc-part", "doc-preface", "doc-prologue",
+    "doc-pullquote", "doc-qna", "doc-subtitle", "doc-tip", "doc-toc", "document",
+    "emphasis", "feed", "figure", "form", "graphics-document", "graphics-object",
+    "graphics-symbol", "group", "heading", "img", "insertion", "list", "listitem", "log",
+    "main", "mark", "marquee", "math", "meter", "navigation", "none", "note",
+    "paragraph", "presentation", "region", "rowgroup", "search", "separator", "status",
+    "strong", "subscript", "superscript", "table", "tabpanel", "term", "time", "timer",
+    "tooltip", "progressbar",
+};
+
+pub static INTERACTIVE_ELEMENT_SCHEMA: &[ElementSchema] = &[
+    ElementSchema {
+        name: "a",
```

Comment:
Storing these tag names multiple times bloats the binary size. We should store these as static values and reference them instead.

Example:
```rs
static TAG_A: &'static str = "a";
static TAG_AREA: &'static str = "area";

ElementSchema {
  name: TAG_AREA,
  attributes: &[]
}
```